### PR TITLE
Store relative paths to variant/$arch/$content_category_name dir

### PR DESCRIPTION
**Pull request by ycheng-aa**
Stare relative paths when importing compose images and compose
rpms. The path is stored with compose, variant, arch, path type.

JIRA: PDC-1358

### CODE REVIEW

File: pdc/apps/compose/models.py [hunk 1]
```diff
@@ -586,3 +586,35 @@ def export(self):
             "url": self.url,
             "synced_content": [item.name for item in self.synced_content.all()]
         }
+
+
+class PathType(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __unicode__(self):
+        return self.type
```
Comment (simozhan) [inline pdc/apps/compose/models.py]: I guess this is a typo, should be self.name, right?

Comment (ycheng-aa) [inline pdc/apps/compose/models.py]: Fixed in f9832de31940767c0b3. Thanks

**Comment by simozhan** [comment]
:+1:

**Comment by bliuredhat** [comment]
looks good for me

**Pull request merged**